spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-bus-config-sample/pom.xml

```diff
@@ -10,12 +10,11 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>spring-cloud-gcp-pubsub-bus-config-sample</artifactId>
-    <groupId>com.google.cloud</groupId>
     <packaging>pom</packaging>
     <name>Spring Framework on Google Cloud Code Sample - Pub/Sub Bus Configuration Management</name>
 
     <properties>
-        <spring-cloud-config.version>3.1.5</spring-cloud-config.version>
+        <spring-cloud-config.version>4.0.0-RC2</spring-cloud-config.version>
```

Comment:
To double check, I'm guessing this will have to be updated to a non-RC version when one is available?

Reply:
Yes, I'll change the version when it's available.

Reply:
Hi. I think is out https://search.maven.org/artifact/org.springframework.cloud/spring-cloud-starter-config/4.0.0/jar

Same for `spring-cloud-dependencies` https://mvnrepository.com/artifact/org.springframework.cloud/spring-cloud-dependencies/2022.0.0

Reply:
@nhmarujo Thanks for the update.

@mpeddada1 version updated.

Reply:
You're welcome! Thank you for your awesome work @JoeWang1127 🙂